### ISSUE TITLE
Question: Using brunch to build a blog

### ISSUE THREAD

**Issue by bapti** [brunch/brunch#970]
Hi,

I've read through all the guides of brunch and it seems great.  I'd thought of using it to build a blog site but I'm struggling to figure out if this is a good use case or not?  I've had a shot with metalsmith too and it's processing steps seem to make more sense to me for slowly manipulating markdown files into the eventual static site.  

However everything written in the guides makes a lot of sense for the use of brunch as it'll be a faster build and better development experience.

I've done a fair bit of googling but not found any examples/skeletons of a workflow for this kind of blog.  I'm not sure if this means that it's not a good use case so I thought I'd ask here first before investing too much more effort.

Thanks
Neil

p.s the guides are brilliant, I followed right through and will definitely be using brunch for my next knockout SPA.

**Comment by moonwave99** [comment]
Hey @bapti , I think that generating a blog/website is beyond Brunch scope...still I used it as the best companion to handle asset generation // distribution for [the generator tool I am developing](https://github.com/moonwave99/waffel) ^^

Unfortunately it is not documented yet, but I am about to publish everything to gh-pages.

I just gathered a minimal setup into a skeleton, [have a look here in case](https://github.com/moonwave99/brunch-with-waffel)!

    brunch new gh:moonwave99/brunch-with-waffel
    brunch w -s

TL;DR yes you can use brunch in a static site generation workflow, I do proficiently.

[sorry for the shameless skeleton promotion]

**Comment by es128** [comment]
@bapti I agree with @moonwave99, this isn't currently within Brunch's scope, although this sort of template composition may be added in the future. There would be ways to accomplish it now using your own custom javascript, but Brunch is intended to build assets for SPA-type of structure which will generally have some form of RESTful backend it relies upon for data.

Tools like docpad, metalsmith, and assemble that label themselves as "static site generators" are more geared toward this task. And brunch-with-waffel looks very interesting too for combining the benefits of Brunch (in its current form) with the static page generation capabilities.

**Comment by bapti** [comment]
Thanks for the input guys, I look forward to seeing your progress on brunch with waffel
